script.js

```diff
@@ -1,4 +1,126 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+/* 
+User input = Scissors/Paper/Stone
+Output = Win/Lose
+    1        2       3        1
+Scissors > Paper > Stone > Scissors
+
+0-0 = 0
+0-1 = -1
+0-2 = -2
+
+1-0 = 1
+1-1 = 0
+1-2 = -1
+
+2-0 = 2
+2-1= 1
+2-2 = 0
+
+ */
+// Gobal Variables
+const selection = ["scissors", "paper", "stone"];
```

Comment:
Great use of array here, making it easy for you to cycle through the possible selections. Not so big in a small game like this, but very useful as the projects you build get bigger and more complex.